docker/Dockerfile

```diff
@@ -41,11 +43,9 @@ RUN yarn global add lerna && \
 # Build
 FROM source${build} AS build
 
-# Download solidity compiler as per: https://github.com/nomiclabs/hardhat/issues/1280#issuecomment-949822371
```

Comment:
I'd preserve this comment as we will have to update this in the future.

Reply:
Done :)